data/json/items/comestibles/dairy.json

```
@@ -302,7 +302,7 @@
     "calories": 85,
     "description": "A ball of white mozzarella cheese.",
     "price": 450,
-    "price_postapoc": 400,
+    "price_postapoc": 80,
```

```suggestion
    "price_postapoc": 60,
```
This is calculated proportionally from calories of meat jerky, which is what docs put as a baseline for item pricing. Technically, by proportions the price is slightly above 61, but mozzarella does spoil twice as fast, so I think rounding it down is acceptable